docker-compose.yml

```diff
@@ -5,7 +5,7 @@ version: '3'
 services:
   db:
     container_name: tsugi_db
-    image: mysql:5.7
+    image: mariadb:10.5
```

Comment:
I think it's up to @csev on whether we have mariadb or mysql:8.0 here. I'm fine with maria.